packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -227,7 +229,9 @@ export const FileSnapshotWriterClassFactory = <TBase extends ReaderConstructor>(
 			}
 
 			for (const blobName of Object.keys(snapshotTree.blobs)) {
-				const buffer = await this.readBlob(snapshotTree.blobs[blobName]);
+				// Non null asserting here, we should use Object.entries() instead
```

Comment:
Originally this was what I was doing but we wanted to non null assert initially so we know we are not breaking anything since we are doing this repo wide